ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -565,7 +603,7 @@ private class DisplayURLLabel: UILabel {
     case .arabic, .hebrew, .persian, .urdu:
       isRightToLeft = true
     default:
-      isRightToLeft = false
+      isRightToLeft = ["http", "https"].contains(URL(string: text)?.scheme ?? "")  // Only left-align if the scheme is not http/https.
```

Comment:
This logic shouldn't go here, which is primarily for determining RTL language modes. Instead you should introduce a new variable that is set when `text` is set that will store whether or not the underlying URL is a web scheme one. You're already doing that when you do `URLOrigin(...).url`, you could store the origin and check the scheme directly instead in the `drawText` override